src/app/sessions/page.test.tsx

```diff
@@ -11,7 +11,13 @@ jest.mock("next/navigation", () => ({
 
 describe("Select Sessions page", () => {
   beforeEach(() => {
-    render(<SelectSessionsPage />);
+    render(
+      <ClientSessionPage
+        firstName="Test"
+        isMember={true}
+        remainingSessions={parseInt("10")}
```

Comment:
curious why you used parseInt here?

Reply:
There was an issue with using remainingSessions=10,  but I just realised that I could just use {10}